src/__swaps__/screens/Swap/hooks/useAnimatedSwapStyles.ts

```diff
@@ -206,8 +207,12 @@ export function useAnimatedSwapStyles({
   });
 
   const searchOutputAssetButtonStyle = useAnimatedStyle(() => {
+    const isPasteMode = outputProgress.value === NavigationSteps.TOKEN_LIST_FOCUSED && !internalSelectedOutputAsset.value;
```

Comment:
extract this condition out since it's used in a few different places.

Reply:
I'd like to but didn't find a way to do it, it's used once in the hook and another in the `SearchInput` component, ideas?